lib/src/footer/Footer.tsx

```diff
@@ -150,11 +143,10 @@ const BottomLinks = styled.div`
   & > span:last-child span {
     display: none;
   }
-  margin: ${(props) => (props.refSize <= responsiveSizes.mobileLarge ? "40px 0 40px 0" : "")};
```

Comment:
Why do you deleted this margin?

Reply:
Because it is adding a lot of space:
![image](https://user-images.githubusercontent.com/34685461/159251451-5e39f0b3-dca6-4ed4-9123-07a821dea4dd.png)

Without margin: 
![image](https://user-images.githubusercontent.com/34685461/159251623-08f4e63b-95cd-4adf-b59b-8f424524c1fc.png)